src/config/hooks.ts

```diff
@@ -106,21 +136,20 @@ export const useModelsSearch = (queryParams = {}) =>
         },
         {
           addQueryPrefix: true,
-        }
+        },
       );
       // cannot use axios https://github.com/miragejs/miragejs/issues/1006
       const response = await fetch(`${MODEL_SEARCH_ENDPOINT}${queryString}`);
```

Comment:
You can build the url using `new URL` and `URLSearchParams`. I personally find it easier than using qs.
```ts
const url = new URL(MODEL_SEARCH_ENDPOINT);
const query = new URLSearchParams({ type: 'models', q: queryParams.q ?? DEFAULT_QUERY });
url.search = query
const response = await fetch(url.toString())
```